src/components/Comments/Comment.js

```diff
@@ -38,6 +46,7 @@ const Comment = ({
   onLongPress,
   updateVote
 }) => {
+  const refSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `refSheet` is a new addition and seems to be used correctly for controlling the `BottomSheetMenu`. It's initialized with `React.useRef()`, which is the correct way to create refs in functional components.
```diff
+ const refSheet = React.useRef();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->